From 2425bc4d969765b284cbe8758182c9a19c3133cf Mon Sep 17 00:00:00 2001
From: Lawrence Rust <lvr@softsystem.co.uk>
Date: Fri, 4 Oct 2013 14:44:52 +0100
Subject: [PATCH 196/207] Transcode: Check aspect ratio when hoonouring the
 cutlist

If the recording is transcoded with a cutlist then the pixel aspect
ratio is currently not set correctly.  This typically affects 4:3
material which is coded at 544x576 resolution.

This patch sets the aspect ratio from the 1st frame decoded after
the cut.

Signed-off-by: Lawrence Rust <lvr@softsystem.co.uk>
---
 mythtv/libs/libmythtv/mythplayer.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/mythplayer.cpp b/mythtv/libs/libmythtv/mythplayer.cpp
index 0a511e6..01020da 100644
--- a/mythtv/libs/libmythtv/mythplayer.cpp
+++ b/mythtv/libs/libmythtv/mythplayer.cpp
@@ -1156,7 +1156,7 @@ void MythPlayer::ReleaseNextVideoFrame(VideoFrame *buffer,
         videoOutput->ReleaseFrame(buffer);
 
     detect_letter_box->Detect(buffer);
-    if (allpaused)
+    if (allpaused || transcoding)
         CheckAspectRatio(buffer);
 }
 
-- 
1.7.9.5

